init.d/autossh

```diff
@@ -45,6 +45,13 @@ start() {
     eend $?
 }
 
+stop() {
```

Comment:
I checked this, and you are right. Default stop() works ok. I added it log time ago and don't know now why. :)
I can try to delete this commit on my side and it should disappear here.